### PR TITLE
1611 cloud openstack

### LINKED ISSUE #1611: /cloud/openstack/ to VBT

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* update /cloud/openstack to vbt

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/cloud/openstack)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/2.%20Cloud?preview=cloud-openstack.png)

## Issue / Card

Fixed #1611

### CODE REVIEW

**Comment by yaili** [comment]
@pmahnke the demo link isn't working

File: static/sass/v1_section/_v1_cloud.scss [hunk 1]
```diff
@@ -0,0 +1,104 @@
+// events on insights news feed, code ported from ../section/_cloud.scss
```
Comment (barrymcgee) [inline static/sass/v1_section/_v1_cloud.scss]: I know this is snowflake code but can it be cleaned up to use BEM and Vanilla spacing variables?

**Comment by pmahnke** [comment]
@yaili  demo - http://www.ubuntu.com-1611-cloud-openstack.demo.haus/cloud/openstack

**Comment by yaili** [comment]
Design review:
- [ ] The text block in hero should align left with grid
- [ ] Not urgent, but is it weird to say "Use …"? Should we say "Try …" instead?
- [ ] Pull quote quotation marks in ubuntu should be orange
- [ ] Items in breadcrumb in S screen are all misaligned
screen shot 2017-06-06 at 13 38 04

- [ ] The image in "Build a production cloud with Autopilot" shouldn't overflow the container at larger screens (screenshot zoomed in to 80% size)
screen shot 2017-06-06 at 13 39 38

- [ ] The image in "Build a production cloud with Autopilot" should probably be hidden at S screen

File: static/sass/v1_section/_v1_cloud.scss (hunk 1 above)
Comment (pmahnke) [inline static/sass/v1_section/_v1_cloud.scss]: I just copied it from Graham... but I fixed it up

**Comment by pmahnke** [comment]
not sure about the breadcrumb.... @barrymcgee ?

**Comment by yaili** [comment]
@pmahnke you removed the design:-1 label… mistake?

**Comment by pmahnke** [comment]
@yaili I updated that [autopilot strip](http://www.ubuntu.com-1611-cloud-openstack.demo.haus/cloud/openstack)... I think it works better now... how is it left to right?

**Comment by barrymcgee** [comment]
Breadcrumbs issue is #1893

**Comment by barrymcgee** [comment]
@pmahnke Should this PR include global-nav files?

**Pull request merged**